content/en/real_user_monitoring/guide/setup-rum-deployment-tracking.md

```diff
@@ -0,0 +1,188 @@
+---
+
+## Overview
+As teams iterate quickly and deploy code, it can be difficult to track down the exact change that caused a spike in errors or slower page load times. RUM Deployment Tracking will enable you to identify when a recent deployment or release is causing performance issues within your application and help you identify the source of the problem. 
+
+## Setup
+You can use the `version` tag to monitor deployments and service behavior in support of your software deployment strategy. To get started with RUM Deployment tracking you will need to add RUM versions to your application.
+
+### Browser RUM
+{{< tabs >}}
+{{% tab "npm" %}}
+
+```javascript
+import { datadogRum } from '@datadog/browser-rum';
+
+// Initialize Datadog Browser SDK
+datadogRum.init({
+  ...
+  version: '1.0.0',
+  ...
+});
+```
+
+{{% /tab %}}
+{{% tab "CDN async" %}}
+
+```html
+<script>
+  (function(h,o,u,n,d) {
+     h=h[d]=h[d]||{q:[],onReady:function(c){h.q.push(c)}}
+     d=o.createElement(u);d.async=1;d.src=n
+     n=o.getElementsByTagName(u)[0];n.parentNode.insertBefore(d,n)
+  })(window,document,'script','https://www.datadoghq-browser-agent.com/datadog-rum-v4.js','DD_RUM')
+  DD_RUM.onReady(function() {
+    DD_RUM.init({
+      ...
+      version: '1.0.0',
+      ...
+    })
+  })
+</script>
+```
+
+{{% /tab %}}
+{{% tab "CDN sync" %}}
+
+```html
+<script src="https://www.datadoghq-browser-agent.com/datadog-rum-v4.js" type="text/javascript"></script>
+<script>
+  window.DD_RUM &&
+    window.DD_RUM.init({
+      ...
+      version: '1.0.0',
+      ...
+    })
+</script>
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+### Mobile RUM
+
+#### Android RUM
+
+The version tag is captured automatically from the application’s manifest
+
+#### iOS RUM
+
+The version tag is captured automatically from the application’s app’s `info.plist`
+
+## Analyze your Deployment Performance in RUM
+
+{{< tabs >}}
+{{% tab "Browser RUM" %}}
+
+### Using version tags on the Application Overview page
+
+An application configured with version tags has a Deployment Tracking section on the Application Overview page. The Deployment Tracking section shows all versions of the application and services that were active during the selected time interval. 
+
+This enables you to adopt modern deployment strategies with peace of mind by letting you quickly roll back release candidates as soon as you spot an issue so you can avoid negative user experiences. These out of the box graphs are aggregated across versions, making it easy to spot problems in your application before they turn into serious issues.
+
+You will see:
+- P75 Loading Time by Version
+- Total User Sessions by Version
+- Error Rate by Version
+
+In the table below these widgets, you will see:
+- The version names deployed for the application and its services over the timeframe.
+- Number of user session for that version
+- Average errors per view
+- P75 Loading Time
+- P75 for Core Web Vitals
```

Comment:
What are we trying to achieve with these bulleted lists? I don't think they are helpful for the user as-is, if they're just stating what's there, they feel more like marketing end less like technical documentation.

Reply:
Yes, this is just stating what's there. I was following a similar pattern to the [APM deployment tracking page ](https://docs.datadoghq.com/tracing/services/deployment_tracking/)but am not opposed to removing this.

Reply:
It looks like on that page they're framed more as "these are the available columns" - I think we could maybe adjust it to be the same if that's what the functionality is, instead of "you will see" as long as is it something that they can add or remove, or are they always there?

Reply:
yes, it's the same functionality as APM. they are always there, users will need to duplicate/copy the widgets to a dashboard to modify them

Reply:
Should we leave this as is then? Or is there some wording change we should make?